examples/downcasting.rs

```diff
@@ -0,0 +1,35 @@
+// In order to downcast from a trait object to a concrete implementation, a trait needs an `as_any` method.
+// [Source](https://stackoverflow.com/a/33687996/6827589)
```

Comment:
You don't need an explicit `Source` link. Simply hyperlink the text above.

Reply:
Addressed in 9ad0699